blocks/data-cards/data-cards.js

```diff
@@ -25,6 +27,92 @@ function createRTEContainer(rteElements) {
   return rteContainer;
 }
 
+/* Carousel Starts here */
+/* eslint-disable no-bitwise */
+
+function arrowIcon(props) {
+  const icon = img();
+  icon.src = `${window.hlx.codeBasePath}/icons/${props}.svg`;
+  icon.alt = `${props}`;
+  icon.loading = 'lazy';
+  icon.dataset.iconName = `${props}`;
+  return icon;
+}
+
+function arrow(props) {
+  const container = p({ class: 'arrow-button-container' });
+  const anchor = button({ class: 'carousel-button' });
+  anchor.classList.add(`button-${props}`);
+  anchor.type = 'button';
+  anchor.append(arrowIcon(props));
+  container.append(anchor);
+  return container;
+}
+
+function moveDirection(itemWidth, option, block) {
+  const carouselItems = block.querySelector('.data-card-items');
+  if (option === '+') {
+    carouselItems.scrollLeft += itemWidth;
+  } else {
+    carouselItems.scrollLeft -= itemWidth;
+  }
+}
+
+function buttonEvents(nextBtn, prevBtn, block) {
+  const cardsContainer = block.querySelector('.data-card-items');
+  const moveRightBtn = block.querySelector(`.button-${nextBtn}`);
+  const moveLeftBtn = block.querySelector(`.button-${prevBtn}`);
+  const screenWidth = window.screen.width;
+  let currentIndex = 0;
+  let maxIndex = cardsContainer.children.length;
+
+  if (screenWidth > 600) {
```

Comment:
why we are considering 600 or 700 px ? mobile viewport we are supporting is below 768px